docs/05_building_block_view.adoc

```diff
@@ -86,46 +88,29 @@ Important Interfaces::
 
 [role="arc42help"]
 ****
-Here you can specify the inner structure of (some) building blocks from level 1 as white boxes.
+Here you can specify the inner structure of (some) building blocks from level 2 as white boxes.
 
-You have to decide which building blocks of your system are important enough to justify such a detailed description.
-Please prefer relevance over completeness. Specify important, surprising, risky, complex or volatile building blocks.
-Leave out normal, simple, boring or standardized parts of your system
+When you need more detailed levels of your architecture please copy this
+part of arc42 for additional levels.
 ****
 
-==== White Box _ArquiSocks_
+==== White Box _Webapp_
 
-image:whitebox_arquisocks.png["Whitebox ArquiSocks"]
+image:whitebox_webapp.png["Whitebox Webapp"]
 
 Motivation::
 
-This diagram shows the relations between the restapi and the webapp. Both building blocks will be explained more detailed in next paragraphs.
+This diagram shows the relations among webapp components and libraries used. The main lirary is mui used for optimize the development of many components. It has been implemented using *React library for user interfaces*.
 
 Contained Building Blocks::
 |===
 |Building block |Description
-| _React_ | Library used in order to improve development of Graphic User Interfaces. 
-| _Webapp_ | Contains the GUI of the application which is useful for clients usability. 
-| _Restapi_ | Contains controllers (routers) in order to manage CRUD requests. Generates responses to the requests.
-
-|===
-
-Important Interfaces::
-|===
-|Interface |Description
-| _Server_ | this element has the responsibility of manage requests of application's clients. These requests come from the webapp.
+| _Api_ | Contains functions to connect with the restapi.
+| _App_ | The main page of the web. Packages (npm): mui.
+| _Components_ | Contains all of the components of the webapp, from the cart to the products list. Packages (npm): mui, inrupt/solid-ui-react.
+| _Contexts_ | It is like the session of the user, it stores products selected.
```

Comment:
![image](https://user-images.githubusercontent.com/79652551/163583618-5baf1636-34c1-4b0d-afe4-27429ec53603.png)
Entonces pondría mejor "allows to share data across the components more easily. For now, the context is sharing selected products in cart".